presto-hive/src/main/java/com/facebook/presto/hive/HdfsConfigurationUpdater.java

```diff
@@ -13,6 +13,7 @@
  */
 package com.facebook.presto.hive;
 
+import com.facebook.hive.orc.OrcFile;
```

Comment:
This is the import for DWRF, not ORC. Use `org.apache.hadoop.hive.ql.io.orc.OrcFile`